### PR TITLE
Cr50

### PR DESCRIPTION
**Pull request by Aurige**
New mode for frozen surface type of transport (CR50 from Norway - accepted)

### CODE REVIEW

**Comment by skinkie** [comment]
@Aurige could you submi the XML-Spy / Oxygen updates separately from this pull request first?

**Comment by Aurige** [comment]
I did a pull request with only XML-Spy / Oxygen updates last week ... but as I work with WML Spy I have to use a branch having these updates. Can you accept my previous XML-Spy / Oxygen  PR, that would make life easier for coming days.

**Comment by skinkie** [comment]
Done.

File: xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd [hunk 1]
```diff
@@ -177,6 +178,7 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="water"/>
 			<xsd:enumeration value="cableway"/>
 			<xsd:enumeration value="funicular"/>
+			<xsd:enumeration value="SnowAndIce"/>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd:181]: What is the reason that this is not "snowAndIce" (lower S)

Comment (skinkie) [reply xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd:181]: image

**Comment by skinkie** [comment]
@Aurige I notice when I look in the current NeTEx this entire ticket is already resolved by @nick-knowles.

**Comment by Aurige** [comment]
@skinkie Yes, this is the case for several of them... I guess we ned to close all such PR

**Comment by skinkie** [comment]
@Aurige only if the documentation is available.

**Pull request closed without merging**